pandas/tests/frame/test_reductions.py

```diff
@@ -1025,6 +1025,46 @@ def test_idxmax_mixed_dtype(self):
         expected = Series([0, 2, 1, 2], index=[1, 2, 3, 4])
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "ID, value, expected_index, expected_value",
+        [([100, 100, 100, 200, 200, 200], [0, 0, 0, 1, 2, 0], [100, 200], [0, 4])],
```

Comment:
Please parametrize over ecpected values and the applied function only

Reply:
thanks @phofl , will redo and push